src/main/resources/static/templates/XXEVulnerability/LEVEL_1/XXE.js

```diff
@@ -14,43 +7,43 @@ let buildXmlRequest = function (
 ) {
   let xmlRequestDocument = document.implementation.createDocument("", "", null);
   let bookElement = xmlRequestDocument.createElement("book");
+
   let bookNameElement = xmlRequestDocument.createElement("name");
-  bookNameElement.innerHTML = bookName;
+  bookNameElement.textContent = bookName; // Use textContent to prevent XSS
   bookElement.appendChild(bookNameElement);
 
   let authorElement = xmlRequestDocument.createElement("author");
-  authorElement.innerHTML = author;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities